lib/checkunusedfunctions.cpp

```diff
@@ -101,7 +101,7 @@ void CheckUnusedFunctions::parseTokens(const Tokenizer &tokenizer, const char Fi
             int scope = 0;
             bool start = true;
             // find all function calls in library code (starts with '(', not if or while etc)
-            while ((scope || start) && markupVarToken) {
+            while ((scope == 0 || start) && markupVarToken) {
```

Comment:
hmm do you intentionally change the behaviour here?

Reply:
Changed `==` to `!=`